src/main/java/common/AbstractSteps.java

```diff
@@ -0,0 +1,25 @@
+package common;
+
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.thucydides.core.steps.ScenarioSteps;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Field;
+
+public class AbstractSteps extends ScenarioSteps {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AbstractSteps.class);
+
+    protected WebElementFacade getWebElementByFieldName(String fieldName, AbstractPage page) {
```

Comment:
Implementation doesn't allow to deal with Page that consists of components or another page (composite page) e.g. sidebars, common header, ....